cmd/dbaas-controller/main.go

```diff
@@ -32,9 +31,9 @@ import (
 )
 
 func main() {
-	if version.Version == "" {
-		panic("dbaas-controller version is not set during build.")
-	}
+	//if version.Version == "" {
+	//	panic("dbaas-controller version is not set during build.")
+	//}
```

Comment:
Why? That should be set by Makefile now, no?

Reply:
Committed accidentally.
It doesn't set ldflags when we call `make run`

Reply:
Please update `install` target to set them then